client/script.js

```diff
@@ -1,30 +1,41 @@
-
 button = document.querySelector('#searchButton');
 searchedTerm = document.querySelector('#search-bar');
+LuckyButton = document.querySelector('#LuckyButton')
 const ul = document.querySelector('ul');
 
 
 
-
 button.addEventListener('click', function SearchResults(event) {
```

Comment:
Potentially better to have this event listener on 'submit' and then your search bar would respond to an enter button press as well as a click on the submit button.